src/components/ChatEntry.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const sender = props.sender;
+  const body = props.body;
+  const timeStamp = props.timeStamp;
+  const likedEvent = props.likedEvent;
+  let like = '🤍';
+  like = !props.liked ? '🤍' : '❤️';
```

Comment:
Great use of a ternary here!  Small suggestion, like can be set directly from the ternary, and then it can be a const variable.
```suggestion
  const like = !props.liked ? '🤍' : '❤️';
```